api/v1alpha1/types.go

```diff
@@ -11,3 +11,13 @@ type UserQuota struct {
 	// max number of buckets the user can create
 	MaxBuckets int `json:"maxBuckets,omitempty"`
 }
+
+type SubUserBinding struct {
+	// name of the subuser
+	// +kubebuilder:validation:Required
+	Name string `json:"name"`
+	// access of the subuser which can be read or write
+	// +kubebuilder:default=read
+	// +kubebuilder:validation:Enum=read;write
```

Comment:
The `// +kubebuilder:validation:Optional` marker is not required when we include the `omitempty` tag. Let's keep the code consistent and use the marker for all optional fields or not use it at all.

Reply:
Sure. Done